README.md

```diff
@@ -45,6 +45,23 @@ To quickly install Nu:
 brew install nushell
 # Windows
 winget install nushell
+# Debian/Ubuntu
+curl -fsSL https://apt.fury.io/nushell/gpg.key | sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/fury-nushell.gpg
+echo "deb https://apt.fury.io/nushell/ /" | sudo tee /etc/apt/sources.list.d/fury.list
+sudo apt update
+sudo apt install nushell
+# RedHat/Fedora/Rocky Linux
+echo "[gemfury-nushell]
+name=Gemfury Nushell Repo
+baseurl=https://yum.fury.io/nushell/
+enabled=1
+gpgcheck=0
+gpgkey=https://yum.fury.io/nushell/gpg.key" | sudo tee /etc/yum.repos.d/fury-nushell.repo
+sudo dnf install -y nushell
+# Alpine Linux
+echo "https://alpine.fury.io/nushell/" | tee -a /etc/apk/repositories
+apk update
+apk add --allow-untrusted nushell
```

Comment:
I would leave them out for now as there are community contributed packages in the official package registries for those, that have signature verification, and I don't want to officially bless setting up unverified channels.
```suggestion
```